python/cudf/cudf/io/csv.py

```diff
@@ -107,6 +107,9 @@ def to_csv(
         df = df.reset_index()
     rows_per_chunk = chunksize if chunksize else len(df)
 
+    if isinstance(path, IOBase):
```

Comment:
Instead of checking the instance does calling `str(path)` work ?

Reply:
I might be missunderstanding - If the path is a file handle, `str(path)` gives you somthing like:

```python
"<_io.TextIOWrapper name='/tmp/pytest-of-rzamora/pytest-86/test_csv_roundtrip0/data-0.csv' encoding='utf-8'>"
```

We could check if path is a string, but that seems less robust than the IOBase check.  that said, I'm certainly open to althernatives.

Reply:
oh right!  I thought this was a path object.  Nevermind